### PR TITLE
Enabled pylint check for mockup objects

### PR DESCRIPTION
**Pull request by IvarsKarpics**
* Added missing imports.
* Fixed import to use absolute imports.
* Removed duplicated methods (for example in the ISPyBClientMockup)
* Fixed PlateManipulatorMockup
* Enable pylint check for mockup directory
* Rewrote MachineInfoMockup to use OrderedDict
* Solved #363 

I could not sort out Oxford700Mockup.py and LimaDetectorMockup.py (unable to import PyTango). So for time beeing I disabled pylint check for these two files.

### CODE REVIEW

File: HardwareObjects/mockup/MachineInfoMockup.py [hunk 1]
```diff
@@ -41,50 +43,50 @@ def __init__(self, name):
...
-        temp_dict["title"] = "Flux"
-        self.values_list.append(temp_dict)
-
-        temp_dict = {}
-        temp_dict["value"] = "-"
-        temp_dict["in_range"] = True
-        temp_dict["title"] = "Disk space"
-        temp_dict["align"] = "left"
-        self.values_list.append(temp_dict)
+        self.values_ordered_dict = OrderedDict()
```
Comment (rhfogh) [inline HardwareObjects/mockup/MachineInfoMockup.py]: This is fine - not sure why it needs to be ordered, though. Do we depend on looping through these records in order somewhere? Anyway, just fine with doing it this way.

File: HardwareObjects/mockup/MachineInfoMockup.py [hunk 2]
```diff
@@ -176,7 +179,7 @@ def sizeof_fmt(self, num):
     def sizeof_num(self, num):
         """Returns disk space formated in exp value"""
 
-        for x in ["m", unichr(181), "n"]:
+        for x in ["m", chr(181), "n"]:
```
Comment (rhfogh) [inline HardwareObjects/mockup/MachineInfoMockup.py]: Will this work in Python2? Trying (in Python 2.7) I get:
>>> chr(181)
'\xb5'
>>> print(chr(181))
�
>>> print(unichr(181))
µ

File: HardwareObjects/mockup/MachineInfoMockup.py (hunk 1 above)
Comment (IvarsKarpics) [reply HardwareObjects/mockup/MachineInfoMockup.py]: We would like to display the parameters in the gui in the ordered way. On the top machine current, status, etc

File: HardwareObjects/mockup/MachineInfoMockup.py (hunk 2 above)
Comment (IvarsKarpics) [reply HardwareObjects/mockup/MachineInfoMockup.py]: If you do that in terminal, yes you will get that. But in the Qt widget it is fine.

Comment (rhfogh) [reply HardwareObjects/mockup/MachineInfoMockup.py]: Cool

File: HardwareObjects/mockup/MachineInfoMockup.py (hunk 1 above)
Comment (rhfogh) [reply HardwareObjects/mockup/MachineInfoMockup.py]: Makes sense

**Comment by IvarsKarpics** [comment]
Ok, after merging the PR #370 I will update this PR and most likely resolve conflicts.

**Comment by IvarsKarpics** [comment]
Hi @rhfogh . I updated pr with the changes introduced with PR #370 . Now it is ready to be merged

**Comment by IvarsKarpics** [comment]
Thanks!

**Pull request merged**